src/stream/src/executor/source/source_backfill_executor.rs

```diff
@@ -206,6 +216,8 @@ impl BackfillStage {
     /// Updates backfill states and returns whether the row backfilled from external system is visible.
     fn handle_backfill_row(&mut self, split_id: &str, offset: &str) -> bool {
         let state = self.states.get_mut(split_id).unwrap();
+        state.num_consumed_rows += 1;
```

Comment:
Now we need to `+=1` since we need to count and persist separately for each split.